src/main/java/at/ac/tuwien/kr/alpha/config/CommandLineParser.java

```diff
@@ -335,16 +335,18 @@ private void handleBranchingHeuristic(Option opt, SystemConfig cfg) throws Parse
 		try {
 			cfg.setBranchingHeuristicName(branchingHeuristicName);
 		} catch (IllegalArgumentException e) {
-			throw new ParseException("Unknown branching heuristic: " + branchingHeuristicName + ". Please try one of the following: " + Heuristic.listAllowedValues());
+			throw new ParseException(
+					"Unknown branching heuristic: " + branchingHeuristicName + ". Please try one of the following: " + Heuristic.listAllowedValues());
 		}
 	}
-	
+
 	private void handleMomsStrategy(Option opt, SystemConfig cfg) throws ParseException {
 		String momsStrategyName = opt.getValue(SystemConfig.DEFAULT_MOMS_STRATEGY.name());
 		try {
 			cfg.setMomsStrategyName(momsStrategyName);
 		} catch (IllegalArgumentException e) {
-			throw new ParseException("Unknown mom's strategy: " + momsStrategyName + ". Please try one of the following: " + BinaryNoGoodPropagationEstimation.Strategy.listAllowedValues());
```

Comment:
Please revert to the old formatting here.

Reply:
done